api/src/api/service.rs

```diff
@@ -75,7 +75,6 @@ impl LambdoApiService {
     }
 
     fn generate_steps(
-        &self,
```

Comment:
I'm curious as to why you made this change; is there a specific reasoning ?

Reply:
This function doesn't depend at all from &self, it doesn't use it and force to instantiate the struct to be able to call this function. 
IMO it's better to have static function whenever that's possible (and easily doable).